src/app/_models/action.ts

```diff
@@ -8,4 +8,6 @@ export interface Action {
     status: string;
     time_created: Date;
     time_modified: Date;
+    object_type: string;
```

Comment:
These 2 fields are enums. I think we can have a type for them to only allow the values coming from the back-end instead of a generic string.

Reply:
I have updated this by adding `enum` in the `actions.ts` let me know if this is what you meant, or if you had another idea in mind.